static/js/robot.js

```diff
@@ -22,12 +22,13 @@ var console=(function(oldCons){
     };
 }(console));
 
+var queryString = location.search;
```

Comment:
This is fine, I generally prefer (style-wise) to be more specific on what gets parsed out of `location.search`, but  this is fine.

Reply:
I don't know if I necessarily resolved this, but I added a comment that hopefully helps